miniapps/meshing/mesh-explorer.cpp

```diff
@@ -32,6 +32,7 @@
 //               mesh-explorer -m ../../data/mobius-strip.mesh
 
 #include "mfem.hpp"
+#include "../common/mfem-common.hpp"
```

Comment:
Can this relative path be handled by the build system?

Reply:
I believe so.

Reply:
I also think it works. The electromagnetic mini apps use such paths and I don't recall any related issues.

Reply:
```suggestion
#include "miniapps/common/mfem-common.hpp"
```
just had a problem with relative paths come up in a different miniapp... granted it was with `mfem.hpp` but it's still probably a good idea not to add more relative path includes

Reply:
Using the path `"miniapps/common/mfem-common.hpp"` breaks out-of-source builds. It is best to revert to the relative path -- we do that in all miniapps that use `mfem-common.hpp`.

Reply:
Interesting, the our-of-source build with CMake worked without issue for me.  Was this with the native make build system?  If so, it sounds like it is missing an `-I` flag.

Reply:
Yes, the issue was with the GNU make build system. We can certainly solve the issue by adding the source MFEM dir with `-I` but the use of the relative path is just more consistent with the other uses of this header.

Reply:
That's fair.  I guess I just see lines like this as one more line that will need to be changed if the including source file is ever moved.